bitsail-connectors/bitsail-connectors-legacy/bitsail-connector-hudi/src/main/java/com/bytedance/bitsail/connector/legacy/hudi/util/RowToAvroConverters.java

```diff
@@ -73,7 +73,19 @@ public Object convert(Schema schema, Object object) {
               return ((Short) object).intValue();
             }
           };
-    } else if (BasicTypeInfo.BOOLEAN_TYPE_INFO.getTypeClass().equals(typeClass) || BasicTypeInfo.INT_TYPE_INFO.getTypeClass().equals(typeClass) ||
+    }
+        else if (BasicTypeInfo.BYTE_TYPE_INFO.getTypeClass().equals(typeClass)) {
+        converter =
```

Comment:
We define check style in [tools/maven/checkstyle.xml](https://github.com/bytedance/bitsail/blob/master/tools/maven/checkstyle.xml), and unfortunately it's not satisfied here.
You can reference [this page](https://bytedance.github.io/bitsail/en/community/contribute.html#checkstyle) to see if checkstyle is satisfied in IDEA.